### PR TITLE
Added meetup type specific routing (fixes pykonik#6)

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: meetpy/meetups/urls.py [hunk 1]
```diff
@@ -6,7 +6,10 @@
 
 urlpatterns = [
     url(r'^spotkania/$', views.MeetupsListView.as_view(), name='meetup_list'),
-    url(r'^(?P<number>\d+)/$', views.MeetupDetailView.as_view(), name='meetup_detail'),
+    url(r'^(?P<meetup_type>[\w-]+)/(?P<number>\d+)/$',
```
Comment (Gandi24) [inline meetpy/meetups/urls.py]: If we update to Django 2, can we switch `?P<name>` to `<str:meetup_type>`?

File: meetpy/meetups/views.py [hunk 2]
```diff
@@ -15,9 +17,45 @@ class MeetupsListView(generic.ListView):
     model = models.Meetup
 
 
-class MeetupDetailView(generic.DetailView):
-    model = models.Meetup
-    slug_url_kwarg = slug_field = 'number'
+class MeetupRedirectOrList(generic.View):
```
Comment (Gandi24) [inline meetpy/meetups/views.py]: Why not TemplateView?

File: meetpy/meetups/urls.py (hunk 1 above)
Comment (artcz) [reply meetpy/meetups/urls.py]: Yes, path() sounds much better, although depends if we're going to merge this or the 2.1 branch first. Either way we can do it later on separate PR when both are merged.

File: meetpy/meetups/views.py (hunk 2 above)
Comment (artcz) [reply meetpy/meetups/views.py]: It's simpler. Using features provided by TemplateView would require overriding `get_context_data` as well, then either assign `meetups` somewhere to `self`, or query again, then call `super().get()`... using `TemplateResponse` directly is simpler and if we're not using things from TemplateView then there's no point in inheriting from it. :)

**Pull request merged**